### PR TITLE
Event & Handler for Pagure PR labels added

### PR DESCRIPTION
**Pull request by jpopelka**
- [x] Tests

### CODE REVIEW

File: packit_service/worker/handlers/pagure_handlers.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        self.project = self.config.get_project(event.project_url)
+
+    def run(self) -> HandlerResults:
+        e = self.event
+        logger.debug(
+            f"Handling labels/tags {e.labels} {e.action.value} to Pagure PR "
+            f"{e.base_repo_owner}/{e.base_repo_namespace}/{e.base_repo_name}/{e.identifier}"
+        )
+        if e.labels.intersection(self.config.pr_accepted_labels):
+            logger.debug(f"About to create a bug @ {self.config.bugzilla_url}")
```
Comment (TomasTomecek) [inline packit_service/worker/handlers/pagure_handlers.py:97]: something's missing here :D but I assume that's the next work, right? to tie this PR and the other PR together

File: packit_service/config.py [hunk 2]
```diff
@@ -64,6 +64,7 @@ def __init__(
         fas_password: Optional[str] = "",
         bugzilla_url: str = "",
         bugzilla_api_key: str = "",
+        pr_accepted_labels: List[str] = None,
```
Comment (TomasTomecek) [inline packit_service/config.py:67]: +1 for making it a list and configurable

File: packit_service/worker/handlers/pagure_handlers.py (hunk 1 above)
Comment (jpopelka) [reply packit_service/worker/handlers/pagure_handlers.py:97]: yes

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
>     * Can you please create also some tests in `tests_requre/database`?

I'd love to, but after upgrading to F32 my ansible hangs on `k8s` module, so I can't run 'make check-inside-openshift'. Filled #651 and will do once resolved. Merging now.

**Pull request merged**